dataproc_jupyter_plugin/controllers/cluster.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2023 Google LLC
```

Comment:
We already have a file in the `controllers` directory for Dataproc called `dataproc`.

Move all of the methods from this file into that one and then delete this entire file.